src/MySQLHandler/MySQLHandler.php

```diff
@@ -94,21 +105,37 @@ private function initialize() {
             $this->pdo->exec('ALTER TABLE `'.$this->table.'` add `'.$c.'` TEXT NULL DEFAULT NULL;');
         }
 
+        // merge default and additional field to one array
+        $this->defaultfields = array_merge($this->defaultfields, $this->additionalFields);
```

Comment:
Why do you override $this->defaultfields here? Btw: IMHO defaultFields would be a better name here. Also I guess you want to write this into $this->fields, was $this->fields is used in line 134 to contruct the statement?